src/styles/rating-indicator.scss

```diff
@@ -228,10 +214,18 @@ $block: #{$fd-namespace}-rating-indicator;
     }
   }
 
-  &--half-star {
+  &.#{$block}--half-star {
     @include fd-rating-indicator-with-half-star($fd-rating-indicator-default-font-size, $fd-rating-indicator-default-spacing-between-icons);
   }
 
+  .#{$block}__label.#{$block}__zero-rating {
+    padding: 0 !important;
+    margin: 0 !important;
+    width: 0 !important;
+    overflow: hidden !important;
+    opacity: 0.0001 !important;
```

Comment:
Please try to achieve this without !important. This element becomes impossible to be modified by end users (in case they need to).